cmd/microcloud/ask.go

```diff
@@ -268,13 +269,10 @@ func (c *initConfig) askLocalPool(sh *service.Handler) error {
 		}
 	}
 
-	// Local storage is already set up on every system.
-	if len(askSystems) == 0 {
-		return nil
-	}
+	// Local storage is already set up on every system, or if not every system has a disk.
+	if len(askSystems) == 0 || len(availableDisks) != len(askSystems) {
+		fmt.Println("Skipping local storage pool setup, some systems do not have an available disk")
```

Comment:
Let's be consistent and use the same scheme as the error for distributed storage which is `No disks available for distributed storage. Skipping configuration` and use this instead:
```suggestion
		fmt.Println("No disks available for local storage. Skipping configuration")
```